frontend/src/app/components/network-graph/network-graph.component.ts

```diff
@@ -206,6 +209,28 @@ export class NetworkGraphComponent implements OnInit, OnChanges {
     this.showGraph();
   }
 
+  ngAfterViewInit() {
+    this.subscription = this.route.queryParamMap
+      .pipe(
+        map(params => params.get('node')),
+        map(nodeAddress =>
+          this._filter(nodeAddress).filter(element => element.type === ElementType.NODE)
+        ),
+        filter(results => results.length === 1),
```

Comment:
So an invalid parameter would lead to an empty list I guess. So it gets simply ignored without any information to the user? Could lead to not indented results if the user expects it to behave according to the parameter, but it just switches unrecognized to the default behavior if the parameter is wrong.
Correct me if this is wrong or simply not relevant.

Reply:
Yes, that's right. Maybe I was a bit lazy.. This query parameter was intended to be used by the WebUI, so there should be no invalid parameter given, but it is always better to have the ability to show an error case to the user. I'll see if I can come up with something easy.